scheduler/cloudformation/cloudformation.go

```diff
@@ -192,6 +193,23 @@ func (s *Scheduler) SubmitWithOptions(ctx context.Context, app *scheduler.App, s
 	return tx.Commit()
 }
 
+func (s *Scheduler) Restart(ctx context.Context, app *scheduler.App, ss scheduler.StatusStream) error {
+	stackName, err := s.stackName(app.ID)
+	if err != nil {
+		return err
+	}
+	output := make(chan stackOperationOutput)
```

Comment:
I think this should be buffered so that the goroutine that sends on this doesn't block forever (runaway goroutines).

```go
make(chan stackOperationOutput, 1)
```